_maps/shuttles/ruin_syndicate_dropship.dmm

```diff
@@ -630,7 +630,7 @@ Tn
 Lq
 YU
 Dx
-zq
+ZI
```

Comment:
is this an issue? i don't know why it renamed it but it seems to be the same tile.

Reply:
no